source/common/upstream/BUILD

```diff
@@ -394,6 +394,41 @@ envoy_cc_library(
     name = "eds_lib",
     srcs = ["eds.cc"],
     hdrs = ["eds.h"],
+    deps = [
+        ":cluster_factory_lib",
+        ":leds_lib",
+        ":upstream_includes",
+        "//include/envoy/config:grpc_mux_interface",
+        "//include/envoy/config:subscription_factory_interface",
+        "//include/envoy/config:subscription_interface",
+        "//include/envoy/local_info:local_info_interface",
+        "//include/envoy/secret:secret_manager_interface",
+        "//include/envoy/upstream:cluster_factory_interface",
+        "//include/envoy/upstream:locality_lib",
+        "//source/common/config:api_version_lib",
+        "//source/common/config:decoded_resource_lib",
+        "//source/common/config:metadata_lib",
+        "//source/common/config:subscription_base_interface",
+        "//source/common/config:subscription_factory_lib",
+        "//source/common/config:utility_lib",
+        "//source/common/config:version_converter_lib",
+        "//source/common/network:address_lib",
+        "//source/common/network:resolver_lib",
+        "//source/common/network:utility_lib",
+        "//source/common/protobuf:utility_lib",
+        "//source/extensions/clusters:well_known_names",
+        "@envoy_api//envoy/api/v2:pkg_cc_proto",
+        "@envoy_api//envoy/config/cluster/v3:pkg_cc_proto",
+        "@envoy_api//envoy/config/core/v3:pkg_cc_proto",
+        "@envoy_api//envoy/config/endpoint/v3:pkg_cc_proto",
+        "@envoy_api//envoy/service/discovery/v3:pkg_cc_proto",
+    ],
+)
+
+envoy_cc_library(
+    name = "leds_lib",
+    srcs = ["leds.cc"],
+    hdrs = ["leds.h"],
```

Comment:
Seems a bit odd that the dependencies of this new leds library match the dependencies of what used to be the eds library.  Is there potential for trimming down this dependency list?